data-raw/data-raw.R

```diff
@@ -0,0 +1,13 @@
+rmfd_guide_shortcuts <- tibble::tribble(
```

Comment:
We could also put the names of the supported packages here. Right now, they're hardcoded in `rmfi_list_packages`

Reply:
Yes, good idea. Will add `rmfd_supported_packages` now, and use it in the function.